### DISCUSSION TITLE
✨ [Feature]: input组件增加聚焦,输入改变等事件

### DISCUSSION THREAD

**Discussion by CatsAndMice** [DevCloudFE/vue-devui#1370] [General]
### What problem does this feature solve

![image](https://user-images.githubusercontent.com/58327088/147400890-2ef04d63-6e6e-4551-850c-28e195c41799.png)
input组件没有事件API。当用户聚焦时，希望可以做一些聚焦后的动作

### What does the proposed API look like

 当鼠标聚焦时，可以触发`onFocus`事件，用户输入时，可以触发`onChange`事件

**Comment by tycsbs** [comment]
同样的归个类, input框也缺少清除的功能, 右侧加个clear的图标

**Comment by kagol** [comment]
赞成，需要加上

**Comment by vaebe** [comment]
<img width="895" alt="截屏2022-10-30 下午5 21 41" src="https://user-images.githubusercontent.com/52314078/198871413-8fe16198-4628-4c08-b6ae-f996ece2f4b7.png">

这里所描述的已经实现 @kagol
